scipy/stats/setup.py

```diff
@@ -27,9 +32,17 @@ def configuration(parent_package='',top_path=None):
     config.add_extension('mvn',
         sources=['mvn.pyf','mvndst.f'],
     )
-        
-    # What do I add here to build Rectangle?
-    
+
+    # Build cython rectangle demo
+    config.add_data_files('rect.pxd')
+    ext = config.add_extension(
+        'rect',
```

Comment:
In general you have less problems when you name the module the same as the source file.  For example, I originally had named the module `rect_demo`, but it was having symbol lookup errors because the source file is named `rect.cxx`. Changing the name resolved the symbol error

Reply:
Which is the module and which is the source?

Reply:
I guess it's technically called an extension and not a module (semantics --  could be that an extension is also a module). The name is first, so you'd import it like this:
```
from scipy.stats.rect import Rectangle
```